src/utils/server.ts

```
@@ -158,12 +159,31 @@ export async function getKYCTokens(scope: "basic" = "basic", redirectURI?: strin
+  if (includeCountryCode) {
+    queryClient.setQueryData(["user", "country"], response.headers.get("User-Country"));
+  }
   if (!response.ok) {
-    const { code } = await response.json();
+    const { code } = (await response.json()) as { code: string };
+    throw new APIError(response.status, code);
+  }
+  return response.json();
+}
+
+export type RampProvider = (typeof RampProvider)[number];
```

Biome (2.3.13)

[error] Shouldn't redeclare 'RampProvider'. Consider to delete it or rename it.

'RampProvider' is defined here:


(lint/suspicious/noRedeclare)